src/overlays/actors/ovl_En_Wiz/z_en_wiz.c

```diff
@@ -601,6 +601,9 @@ void EnWiz_SelectPlatform(EnWiz* this, PlayState* play) {
                 break;
 
             case EN_WIZ_FIGHT_STATE_SECOND_PHASE_CUTSCENE:
+                //! @bug: Setting the Wizrobe's position to the first platform *without* updating
+                //! this->curPlatformIndex can cause a bug later in EnWiz_Damaged. To fix this,
+                //! set this->curPlatformIndex to 0 here.
```

Comment:
I think I prefer this way of wording it
```suggestion
                //! @bug: Setting the Wizrobe's position to the first platform *without* updating
                //! this->curPlatformIndex can cause a bug later in EnWiz_Damaged. A way to fix this is
                //! setting this->curPlatformIndex to 0 here.
```

Reply:
I like your way of wording it too, though I tweaked it just a bit.